README.md

```diff
@@ -260,7 +260,7 @@ using `cmake -D BUILD_SHARED_LIBS=OFF ..`.
 | - none     | ✓       | ✓     | ✓     | ☑       | ☑   |
 |            |         |       |       |         |     |
 | Backends   |         |       |       |         |     |
-| - inproc   | ✓       | ✓     | ✓     | ☑       |     |
+| - inproc   | ✓       | (✓)*  | ✓     | ☑       |     |
```

Comment:
nit: is there a reason why we keep `inproc` as the first listed item if macOS doesn't really support it? Maybe we should move the default `crashpad` to the top of this list; as a user that would be less confusing imo